Sources/SwiftInspectorCommands/Tests/PropertyAnalyzerCommandSpec.swift

```diff
@@ -87,7 +87,7 @@ final class PropertyAnalyzerCommandSpec: QuickSpec {
         }
 
         it("fails when path is empty") {
-          let result = try? TestTask.run(withArguments: ["properties", "--name", "Some"])
+          let result = try? TestTask.run(withArguments: ["properties", "--name", "Some", "--path", ""])
```

Comment:
This test was not testing what it said, we were never passing an empty path here.

Reply:
same q here: could we assert that the thrown error is what we expect?